agent/connect/parsing.go

```diff
@@ -111,13 +113,35 @@ func KeyId(raw interface{}) ([]byte, error) {
 		return nil, err
 	}
 
-	// String formatted
 	kID := sha256.Sum256(bs)
-	return []byte(strings.Replace(fmt.Sprintf("% x", kID), " ", ":", -1)), nil
+	return kID[:], nil
```

Comment:
I didn't know what this is doing, but https://blog.golang.org/go-slices-usage-and-internals knows:

>  This is also the syntax to create a slice given an array:

```
x := [3]string{"Лайка", "Белка", "Стрелка"}
s := x[:] // a slice referencing the storage of x
```